pkg/systemd/terminal.jsx

```diff
@@ -186,6 +268,22 @@ const _ = cockpit.gettext;
                             </ToolbarContent>
                         </Toolbar>
                     </div>
+                    <div className="ct-terminal-dir-alert">
+                        {this.state.pathError && <Alert isInline variant='danger'
+                        title={cockpit.format(_("Error opening directory: $0"), this.state.pathError)}
+                        actionClose={<AlertActionCloseButton onClose={this.dismissError} />} />}
+                        {this.state.changePathBusy && <Alert title={_("There is still a process running in this terminal. Changing the directory will kill it.")}
+                            variant="warning"
```

Comment:
*This* on the other hand *should* be "danger", as killing a running process may lead to some data loss.